python/lsst/meas/base/plugins.py

```diff
@@ -68,6 +71,103 @@
                     TransformClass=bl.ApertureFluxTransform, executionOrder=BasePlugin.FLUX_ORDER)
 
 # --- Single-Frame Measurement Plugins ---
+class SingleFrameVarianceConfig(SingleFramePluginConfig):
+        scale = lsst.pex.config.Field(dtype=float, default=5.0, optional=True,
+                                      doc="scale factor to apply to shape for aperture")
+        mask = lsst.pex.config.ListField(doc="mask planes to ignore", dtype=str,
+                                         default=["DETECTED", "DETECTED_NEGATIVE"])
+
+@register("base_Variance")
+class SingleFrameVariancePlugin(SingleFramePlugin):
```

Comment:
See my note above about forced measurements. It is possible to create them, but that would belong in a different ticket for creating something new vs porting from HSC